tests/cypress/e2e/results.cy.js

```diff
@@ -78,6 +79,51 @@ config?.pages?.forEach((page, _i) => {
 		});
 
 		describe('Pagination', () => {
+
+			it("loads next page when scrolled while using Infinite Scroll", function () {
```

Comment:
I think this needs more consideration.

What if the site is using a "load more" button?
Do we really have to reload the entire page and re-add the local bundle? Like the screenshot part of this test, it is much more reliable to not have to reload everything.